### PR TITLE
Localize TooltipText

### PR DESCRIPTION
**Pull request by TimOPI**

### CODE REVIEW

**Comment by TimOPI** [comment]
Including Help topic localization in this pull request.

**Comment by TimOPI** [comment]
@rcoop017 FYI, I think I have one or two help topics left to localize.  Tooltip text is all done.  Both are covered in this pull request.

File: src/components/help/topics/QualifyHomeless.jsx [hunk 1]
```diff
@@ -1,31 +1,64 @@
...
+              defaultMessage="are living in cars, parks, public spaces, abandoned buildings, substandard housing, bus or train stations, or similar settings."
+            />
             </li>
           </ul>
         <p>
-        	If you believe children in your household meet one or more of these descriptions and you haven’t been told your children will get free meals, please contact {organization.name}.
+        <FormattedMessage
+              id="help.topic.QualifyHomeless.answer6"
+              description="Answer for the qualification of homeless students topic."
+              defaultMessage="If you believe children in your household meet one or more of these descriptions and you haven’t been told your children will get free meals, please contact {organization.name}."
```
Comment (ambit-developer) [inline src/components/help/topics/QualifyHomeless.jsx]: Variable should be `organizationName `

**Comment by TimOPI** [comment]
I’ll see if I can figure that out!

**Comment by TimOPI** [comment]
@ambit-developer Fixed the variable name.  Good catch.  Also, got my tab changes set up thanks to @rcoop017

**Pull request merged**